### PR TITLE
Completed express-mutations exercise. - Jody Clark

### PR DESCRIPTION
**Pull request by jclark-14**
express-mutations-1
express-mutations-2

### CODE REVIEW

File: express-mutations/server.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      throw new ClientError(400, 'lastName is required');
+    }
+    const sql = `
+    insert into "actors" ("firstName", "lastName")
+    values ($1, $2)
+    returning *
+    `;
+    const params = [firstName, lastName];
+    const results = await db.query(sql, params);
+    const actor = results.rows;
```
Comment (robgmerrill) [inline express-mutations/server.ts:54]: Did this return an object or an array with an object in it? I believe that we only want the object.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
One small change requested.

File: express-mutations/server.ts (hunk 1 above)
Comment (jclark-14) [reply express-mutations/server.ts]: Correction made.

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**